### DISCUSSION TITLE
How can Increase the padding of the selection box ?

### DISCUSSION THREAD

**Discussion by gitHoussem** [bcakmakoglu/vue-flow#1635] [Q&A] [answered]
Hi,
When performing a multi selection using `shift`, we get a blue box that fits the corners of the nodes perfectly. 

<img width="591" alt="image" src="https://github.com/user-attachments/assets/40178cab-3ead-4671-abb4-ca1b48cc9ef8">

Is there a way to make the box a little bit bigger (similar to a padding) so that there is "room" between the corner of the blue box and the corner of the node ? 

Thank you !

**Comment by bcakmakoglu** [comment, marked as answer]
This is not supported at this point.
I'll have to discuss if this is something that could be added at some point but for now it's not possible to add padding to the box.

**Reply by gitHoussem** [reply]
I appreciate the quick response. Thanks again

**Reply by sefgit** [reply]
a simple workaround would be:

.vue-flow__nodesselection-rect {
    transform: scale(1.1);
}
